include/bluetooth/bluetooth.h

```diff
@@ -517,6 +517,37 @@ enum {
 	 * @note Requires @ref BT_LE_ADV_OPT_USE_NAME
 	 */
 	BT_LE_ADV_OPT_FORCE_NAME_IN_AD = BIT(18),
+
+	/**
+	 * @brief Advertise GATT service UUIDs.
+	 *
+	 * Include the registered GATT service UUIDs automatically when
+	 * advertising. The UUIDs will be placed in the advertising data if
+	 * possible, otherwise it will be moved to the scan response data.
```

Comment:
it -> they ?

Also: Will they all be moved, or just the ones that do not fit (so that they are split between the two packets)?